apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/tag/Tags.java

```diff
@@ -102,6 +102,8 @@ private Tags() {
 
     public static final String VAL_LOCAL_SPAN_AS_LOGIC_ENDPOINT = "{\"logic-span\":true}";
 
+    public static final StringTag JSON_RPC_METHOD = new StringTag(1, "jsonrpc.method");
```

Comment:
We usually only put general tags into `apm-agent-core` module, maybe you should consider moving this tag into a `Constants` util class in the `jsonrpc4j` module.

Reply:
fixed

Reply:
> We usually only put general tags into `apm-agent-core` module, maybe you should consider moving this tag into a `Constants` util class in the `jsonrpc4j` module.

fixed